_includes/event/melbourne/prizes.html

```diff
@@ -35,36 +39,118 @@ <h4>
+          </div>
+        </div>
+      </div>
+      <div class="prize col-md-6">
+        <div class="prize-card">
+          <div class="prize-card-inner">
+            <img src="/img/sponsors/xero.png" alt="Xero" class="uh-sponsor-logo"/>
+            <h4>Xero #beautiful Award</h4>
+            <p class="brief">Awarded to the team that align with Xero's #beautiful values (they are #human, #challenge, #champion, #ownership and, of course, #beautiful).</p>
             <p>To be announced</p>
           </div>
         </div>
       </div>
-      <div class="prize col-md-4">
+
+      <!--<div class="prize col-md-4">
         <div class="prize-card">
           <div class="prize-card-inner">
-            <h4>Most Creative Idea</h4>
+            <h4>Bosch </h4>
             <p class="brief">Awarded to the project with the most original idea or reinvention.</p>
-            <p>To be announced</p>
+            <p>To be announced</p
+            <img src="/img/sponsors/bosch.png" alt="Xero" class="uh-sponsor-logo" />
+          </div>
+        </div>
+      </div>-->
+    </div>
+
+    <div class="row">
+      <div class="prize col-md-4">
+        <div class="prize-card">
+          <div class="prize-card-inner">
+            <img src="/img/sponsors/ittybittyapps.png" alt="ittybittyapps" class="uh-sponsor-logo"/>
+            <h4>Best Mobile Experience</h4>
+            <p class="brief">Awarded to the app that can produce the best user design and experience on mobile devices!</p>
+            <p>Anker 20100mAh 2-Port Portable Power Bank!</p>
+          </div>
+        </div>
+      </div>
+      <div class="prize col-md-4">
+        <div class="prize-card">
+          <div class="prize-card-inner">
+            <img src="/img/sponsors/nab-labs.png" alt="NAB Labs" class="uh-sponsor-logo"/>
+            <h4>NAB Labs Pitch Experience</h4>
+            <p class="brief">This prize will give the chosen team a chance to come to NAB Labs and take part in our Innovation Challenge. The team will get to pitch to a panel of senior judges, and if successful will be able to experiment with Labs on the initiative.</p>
+            <p>A spot in NAB Lab's Innovation Challenge</p>
+          </div>
+        </div>
+      </div>
+      <div class="prize col-md-4">
+        <div class="prize-card">
+          <div class="prize-card-inner">
+            <img src="/img/sponsors/tech-domains.png" alt="Get Tech" class="uh-sponsor-logo"/>
+            <h4>Best Hack on a .tech Domain</h4>
+            <p class="brief">Awarded to the best hack/website submitted on a .TECH domain</p>
+            <p>$250 Amazon Gift Vouchers</p>
```

Comment:
Maybe "$250 Amazon Gift Voucher for each team member" to be consistent with the rest of the page?

Reply:
Not sure if it's for each team member